app/components/common/button/spinner.tsx

```diff
@@ -8,9 +8,29 @@ const s = require('./spinner.scss');
 interface ButtonSpinnerProps {
   color: string;
   size: ButtonSize;
+  variant: string;
+  disabled?: boolean;
   className?: string;
 }
-const ButtonSpinner: React.FC<ButtonSpinnerProps> = ({ className, size, color = 'white' }) => {
+
+function getSpinnerColor(buttonColor: string, variant: string, disabled?: boolean) {
+  if (disabled) return '#bbc2d0';
+
+  if (variant === 'contained' && buttonColor === 'blue') return 'white';
+
+  switch (buttonColor) {
+    case 'blue':
+      return '#3e7fff';
```

Comment:
you can import SCSS variable value.
the example is in `app/components/mobilePaperShow/mobilePaperShow.tsx:29`